task/tkn-bundle/0.1/README.md

```diff
@@ -1,51 +1,23 @@
-# tkn-bundle - Tekton Task to push a Tekton Bundle to an image registry
+# tkn-bundle task
 
-Tekton Task to build and push Tekton Bundles (OCI images) which contain
-definitions of Tekton objects, most commonly Task and Pipeline objects.
+Creates and pushes a Tekton bundle containing the specified Tekton YAML files.
 
-Task finds all `*.yaml` or `*.yml` files within `CONTEXT`, packages and pushes
-them as a Tekton Bundle to the image repository, name and tag specified by the
-`IMAGE` parameter.
-
-## Input Parameters
-
-The task supports the following input parameters.
-
-| Name    | Example                 | Description                              |
-|---------|-------------------------|------------------------------------------|
-| IMAGE   | registry.io/my-task:tag | Reference of the image task will produce |
-| CONTEXT | my-task/0.1             | Paths to include in the bundle image     |
-| HOME    | /tekton/home            | Value for the HOME environment variable  |
-
-`CONTEXT` can include multiple directories or files separated by comma or space.
-Paths can be negated with exclamation mark to prevent inclusion of certain
-directories or files. Negated paths are best placed at the end as they operate
-on collected paths preceeding them. For example if `CONTEXT` is set to
-`"0.1,!0.1/spec"` for this tree:
-
-    .
-    ├── 0.1
-    │   ├── README.md
-    │   ├── spec
-    │   │   ├── spec_helper.sh
-    │   │   ├── support
-    │   │   │   ├── jq_matcher.sh
-    │   │   │   └── task_run_subject.sh
-    │   │   ├── test1.yaml
-    │   │   ├── test2.yml
-    │   │   ├── test3.yaml
-    │   │   └── tkn-bundle_spec.sh
-    │   ├── TESTING.md
-    │   └── tkn-bundle.yaml
-    └── OWNERS
-
-Only the `0.1/tkn-bundle.yaml` file will be included in the bundle.
```

Comment:
It would be a shame to lose this description, we can probably ignore what the generate-readme script wants to do here

Reply:
Yeah, bit lazy on my side...